ui/shared/AccountSelector.qml

```diff
@@ -22,10 +20,38 @@ Item {
     property string showAssetBalance: ""
     property int dropdownWidth: width
     property alias dropdownAlignment: select.menuAlignment
+    property bool isValid: true
+    property var reset: function() {}
+
+    function resetInternal() {
+        accounts = undefined
+        selectedAccount = undefined
+        isValid = true
```

Comment:
Is it expected that it's valid by default?

Reply:
Yes, there is no state in which a selected account is not valid.